Assets/AppCenter/Plugins/iOS/Crashes/CrashesUnity.h

```diff
@@ -20,5 +20,8 @@ extern "C" void* appcenter_unity_crashes_last_session_crash_report();
 extern "C" void appcenter_unity_crashes_set_user_confirmation_handler(bool(* userConfirmationHandler)());
 extern "C" void appcenter_unity_crashes_notify_with_user_confirmation(int userConfirmation);
 extern "C" void appcenter_unity_start_crashes();
+extern "C" void* app_center_unity_crashes_get_error_attachment_log_text(char* text, char* fileName);
+extern "C" void* app_center_unity_crashes_get_error_attachment_log_binary(unsigned char* data, int size, char* fileName, char* contentType);
+extern "C" void* app_center_unity_create_error_attachments_array(MSErrorAttachmentLog* log0, MSErrorAttachmentLog* log1);
```

Comment:
maybe you should name them the same as in definition (errorAttachment0 .. 1)

Reply:
Good idea, thank you